runtime/reconcilers/model.go

```diff
@@ -314,6 +314,13 @@ func (r *ModelReconciler) Reconcile(ctx context.Context, n *runtimev1.ResourceNa
 		} else {
 			model.State.TotalExecutionDurationMs = model.State.LatestExecutionDurationMs
 		}
+
+		// Run model tests if no partitions are configured
```

Comment:
This will also run if partitions are configured (this code path is shared for both partitioned and non-partitioned models, right?)